syntaxes/elm.json

```diff
@@ -343,17 +343,17 @@
       ]
     },
     "char": {
-      "name": "char.quoted.elm",
+      "name": "string.quoted.char.elm",
       "begin": "'",
       "beginCaptures": {
         "0": {
-          "name": "punctuation.definition.char.begin.elm"
+          "name": "punctuation.definition.string.begin.elm"
```

Comment:
Please don't change these two

Reply:
Any particular reason for using `char` over `string`? They're characters indeed, but the scopes for characters in C, C++ and Go all use `punctuation.definition.string.{begin,end}.<language>`.

Not that it matters anyway, it works fine regardless.

Reply:
If you go for string, it can't be highlighted differently due to the naming being the same I think. So this is just enabling themes to highlight `'` differently if they really want to.

Reply:
Oh, okay.